ui/src/app/shared/services/requests.ts

```diff
@@ -27,7 +26,7 @@ let baseHRef = '/';
 const onError = new BehaviorSubject<agent.ResponseError>(null);
 
 function toAbsURL(val: string): string {
-    return path.join(baseHRef, val);
+    return `${baseHRef.replace(/\/+$/, '')}${val}`;
```

Comment:
I'm curious about possible security implications for this mangling.
Is there some standard lib function we can use instead?

Reply:
done

The logic of this code is to remove unnecessary '/' at the end of the string, so I used lodash to handle it.

```
`${_.trimEnd(baseHRef, '/')}${val}`
```